### DISCUSSION TITLE
Plants Talk 【 Xiaoyi Xie & Yutong He】

### DISCUSSION THREAD

**Discussion by xiaoyi-xie** [realitydeslab/playshop-summer-2024#11] [Proposal]
ChineseTitle：植物说话
**Research Questions:**
when humansare able to communicate with plants, wilhe personified plants inspire empathy in humans to pulants on an equalfooting?

**Keywords:**
Artificialintelligence;machine translation; plants;empathy

**Artefact / Media:**
 Video

**Abstract:**
In a future filled with all kinds of technological data, humarbeings are paying less and less attention to benign interaction with nature,neglecting communication with natureand putting human self-consciousness on a high pedesta.
Based on the fact that plants have beenproven to beable to emit screams.there are existing learing ma.chines that can judge the environmental conditions oplants through their screams. Nowadays, with the advenof Allanguagemodels，Acan translatepetween mutiple languages,enabling cross-linguistic communication.
Ne hvnothesise that oase0CnAs earnind mechanIsm!for translating human-to-human languagedels, as welas existing technologies for accommodating plant languag.es,a database of plant lanquages could be created and Acould be used to translate plant languages to human lan.guages. At the same time, we reflect on the question: whehumans can communicate with plants, can the personifiecplants stimulate the empathy of humans to put plants on arequal footing?

**References:**
[1] Nie Zhenzhao.2023.On machine translation in thescientifc age
[2] Lin Meirong. 2024. Construction and application oftranslation Al agent project based on large language model
[3] Li Yachao, Xiong Deyi, Zhang Min. 2017. A review ofneural machine translation
[4]Science and Technology Daily. Hualing. 2022. Withthe help of Al, humans can understand the “joys, angersorrows and happiness’ of animals
[5]Wang Wangi. 2018. Research on the current status andprospects of artifcial intelligence applications in thelanguage service industry
[6]Wang Haifeng.Wu Hua.He Zhongjun.Huang Li-ang.kenneth ward church.Progress in MachineTranslation[J].Engineering,2022,18(11):143-153
[7]Michael Jensen.University of Copenhagen - FacultyofScience.2022.Pig grunts reveal their emotions
[8] Plant electrical communication.https://www.youtube.com/watch?v-pvBlSFVmoaw
[9] https://www.youtube.com/watch?v=2tgdCB3v7mE
[10] https://www.youtube.com/watch?v=e- AGgOJ3VA
[11] https://www.youtube.com/watch?v=6g4-PEzqnhw&t=23s
[12] Green symphonies: a call for studies on acoustic communication in plants.Monica Gagliano .25 November2012
[13] Sounds emitted by plants under stress are airborne andinformative:Itzhak Khait.Ohad Lewin-Epstein.RazSharon, VOLUME 186,ISSUE 7,P1328-1336.E10,MARCH 30,2023
[14] Plants Emit Sounds- Especially When Stressed,TelAviv University
[15] In groundbreaking research, Tel Aviv U team recordsplants 'talking’ for first time: Sue Surkes, 30 March2023
[16]  The flowering of plant bioacoustics: how and why:MonicaGagliano ，0lApril2013
[17] Application of plant audio control technology ingreenhouse vegetable production: Hou Tianzhen.LiBaoming, Teng Guanghui, Zhou Qing, Xiao YingpingOi Lirong 2009.25

**Comment by xiaoyi-xie** [comment]
<img width="854" alt="1" src="https://github.com/realitydeslab/playshop-summer-2024/assets/173814507/ebe7ae6f-2147-4c46-8b47-fd913683ec3e">
<img width="560" alt="微信图片_20240627143235" src="https://github.com/realitydeslab/playshop-summer-2024/assets/173814507/6fe87b0f-bfa9-424d-8b1d-633eb0b063bc">

**Comment by glitch-kraft** [comment]
https://en.wikipedia.org/wiki/Data_set

**Reply by glitch-kraft** [reply]
https://medium.com/@vidyasagar994356/18-online-resources-for-efficient-dataset-collection-b83e423d7ecf

**Comment by glitch-kraft** [comment]
**Speaking Plants**

https://weather.com/science/nature/video/this-is-what-it-sounds-like-when-stressed-plants-scream
https://www.ndtv.com/science/plants-scream-when-uprooted-scientists-capture-sound-for-first-time-5373008
https://www.youtube.com/watch?v=nIXNgxL9ybs

**Comment by glitch-kraft** [comment]
https://www.hackster.io/alien_again/plant-translator-4d58e8
https://www.youtube.com/watch?v=dINEHm1aczY

**Comment by glitch-kraft** [comment]
https://www.cell.com/action/showPdf?pii=S0092-8674%2823%2900262-3

https://english.tau.ac.il/plants_emit_sounds

**Comment by xiaoyi-xie** [comment]
https://www.ecinnovations.com.cn/blog/ai-principles-applications-and-development-of-machine-translation/

https://www.cell.com/heliyon/fulltext/S2405-8440(24)04137-9

**Comment by xiaoyi-xie** [comment]
http://www.cqvip.com/qk/70988x/201609/670034590.html 
https://www.timesofisrael.com/in-groundbreaking-research-tel-aviv-u-team-records-plants-talking-for-first-time/ 
https://www.ndtv.com/science/on-camera-scientists-capture-plants-talking-to-each-other-for-the-first-time-4916301 
https://medium.com/@chasegison/how-ai-is-transforming-content-localization-and-translation-caf3da929c1 
https://edition.cnn.com/2023/03/30/world/plants-make-sounds-scn/index.html 
https://blog.csdn.net/guoyuhaoaaa/article/details/54984636 
https://en.wikipedia.org/wiki/Neural_machine_translation
https://towardsdatascience.com/neural-machine-translation-15ecf6b0b
https://www.ecinnovations.com.cn/blog/ai-principles-applications-and-development-of-machine-translation/
https://www.cell.com/heliyon/fulltext/S2405-8440(24)04137-9
https://en.wikipedia.org/wiki/Data_set
https://medium.com/@vidyasagar994356/18-online-resources-for-efficient-dataset-collection-b83e423d7ecf
https://weather.com/science/nature/video/this-is-what-it-sounds-like-when-stressed-plants-scream
https://www.ndtv.com/science/plants-scream-when-uprooted-scientists-capture-sound-for-first-time-5373008
https://www.youtube.com/watch?v=nIXNgxL9ybs
https://www.hackster.io/alien_again/plant-translator-4d58e8
https://www.youtube.com/watch?v=dINEHm1aczY
[https://www.cell.com/action/showPdf?pii=S0092-8674%2823%2900262-3](https://www.cell.com/action/showPdf?pii=S0092-8674(23)00262-3)
https://english.tau.ac.il/plants_emit_sounds

**Comment by xiaoyi-xie** [comment]
当人与植物语言能够互相转译后，人格化的植物是否能激起人类将植物置于同等地位的同理心?
whether anthropomorphic plants can inspire humans to place plants on an equal footing after humans and plants can translate each other's languages.



在各种科技数据充斥的未来，人类越来越不重视与自然的良性互动，将人类自我意识高高挂起，忽视与自然的沟通，基于植物能够发出尖叫已被证实，学习机器能够通过植物尖叫判断植物所处的环境状况。
In the future filed with a kinds of scientific and technological data, humans pay less and less attention to the posive interaction with naturput human self-consciousness on the back burner, and ignore the communication with nature.Based on artificjal inteligence's learning of human daily language and the behavioral patterns behind it,and the fact that plants can scream, the learing machine can judge the environmental conditions of plants through plant screams.

我们想做的是基于原有的ai机器对于人类之间语言模型翻译的学习机制，推测ai将植物语言和人类语言转译的过程，包括如何收集植物尖叫的数据、更多可能使植物尖叫的方式，以及推测技术通过大量的训练，学习从源语言到目标语言的映射关系。在翻译时，ai根据这些映射关系翻译成目标语言文本。
What we want to do is to use the existing AI machine learning mechanism for human language model translation to speculate the process of AI translating plant language and human language, including how to collect data on plant screams, more possible ways to make plants scream, and speculate that the technology learns the mapping relationship from the source language to the target language through a lot of training. During translation, AI translates into the target language text based on these mapping relationships.
我们接下来主要会阅读更多研究植物发声以及机器翻译的原理相关文章。
最后我们会以视频的方式来呈现这个过程。
Next, we will mainly read more articles on the research of plant vocalization and the principles of machine translation.
Finally, we will present this process in the form of a video.

**Comment by xiaoyi-xie** [comment]
![预告关键帧](https://github.com/realitydeslab/playshop-summer-2024/assets/173814507/8592aa60-2ed7-406d-a9d6-3d84b6236b20)
Keyframe

**Comment by xiaoyi-xie** [comment]
Description: In the park, researchers in white clothes are listening to the sounds of plants with special microphone equipment.

**Comment by glitch-kraft** [comment]
The plant vocal dataset was successfully collected. This enabled the creation of the language library and translation using natural language processing and machine learning. The first plant press conference will be held this evening.

**Reply by glitch-kraft** [reply]
The plant's vocal dataset, language library and translation model will be completed using natural language processing and machine learning. The first plant press conference will be held tonight.
